resource_files/wind/35.2018863_-101.945027_nasa_2012_60min_80m.srw

```diff
@@ -0,0 +1,8789 @@
+035d20N-101d95W,n/a,n/a,n/a,2012,35.2018863,-101.945027,1097.1

```

Comment:
I believe this was intentional. Since we added tests for downloading and reading resource files for nasa, mirroring the nsrdb/windtoolkit tests, these files are specific to the nasa tests for the TX location lat/lon in test_resource_download.py. There are equivalent files being tracked in HOPP master for the nsrdb/windtoolkit tests (_psmv3_ and _windtoolkit_ instead of _nasa_)